lib/order.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'menu'
+require_relative 'text'
+
+class Order
+  attr_reader :current_order, :menu
+
+  def initialize(menu = Menu.new)
+    @current_order = {}
+    @menu = menu
+  end
+
+  def select_item(item,quantity)
+    if item_included?(item)
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)